capstone-api/models/User.js

```diff
@@ -114,7 +112,7 @@ class User {
     let followers = await this.getFollowers(username);
     let result = [];
     for (let i = 0; i < followers.length; i++) {
-      let userPosts = await this.timeline(followers[i]);
+      let userPosts = await this.getTimeline(followers[i]);
       result = result.concat(userPosts);
     }
```

Comment:
super nit that you prob don't need to implement in this PR, but in general in javascript the map function tends to be more used than for loops. I think this is a good case for using the mapping function. it is a bit easier to read too

Reply:
That's a good point I have several areas in my code where I have for loops. Will add to my list to do when everything is in one branch